openedx/core/djangoapps/oauth_dispatch/dot_overrides/views.py

```diff
@@ -94,4 +95,4 @@ def get(self, request, *args, **kwargs):
             return self.render_to_response(self.get_context_data(**kwargs))
 
         except OAuthToolkitError as error:
-            return self.error_response(error)
+            return self.error_response(error, application)
```

Comment:
Reviewer: FYI: This was blowing up without this required argument when I had scope problems.